docs/upgrade.html

```diff
@@ -44,6 +44,8 @@ <h5><a id="upgrade_400_notable" href="#upgrade_400_notable">Notable changes in 4
                         <li>The <code>kafka.common.requests.DescribeLogDirsResponse.ReplicaInfo</code> class was removed. Please use the <code>kafka.clients.admin.DescribeLogDirsResult.descriptions()</code> class
                             and <code>kafka.clients.admin.DescribeLogDirsResult.allDescriptions()</code>instead.
                         </li>
+                        <li>The <code>kafka.common.errors.NotLeaderForPartitionException</code> class was removed. Please use the <code>kafka.common.errors.NotLeaderOrFollowerException</code> class instead.
```

Comment:
trunk is in 4.1.0 now, so please create the new section :)

Reply:
> trunk is in 4.1.0 now, so please create the new section :)

Thanks! Updated it.

Reply:
It seems we have the room to push more cleanup to 4.0.0, so could you please move it back to 4.0.0 section? I will backport this to 4.0 branch

Reply:
> It seems we have the room to push more cleanup to 4.0.0, so could you please move it back to 4.0.0 section? I will backport this to 4.0 branch

Sure.I will move it back.